sitemedia/scss/base/_fonts.scss

```diff
@@ -2,25 +2,68 @@
 // Project fonts, including font face declarations and font stacks.
 // -----------------------------------------------------------------------------
 
-// If not using Google Fonts, @font-face declarations would go here. Since we
-// use Google Fonts, we instead use the CSS2 API and request all the families,
-// weights, and variants we need using a single @import. See:
-// https://developers.google.com/fonts/docs/css2
+@charset "UTF-8";
 
-// Merriweather: light, light italic, regular, regular italic
-@import url("https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;1,300;1,400&display=swap");
+// Usage counter and license for myfonts
+@import url("//hello.myfonts.net/count/4098e1");
```

Comment:
one more thought — in the past we've made the counter a production-only behavior, on the assumptions that dev/test requests shouldn't count; any thoughts?